source/Calamari/Kubernetes/SetupKubectlAuthentication.cs

```diff
@@ -430,10 +448,10 @@ bool TrySetAws()
 
         void CreateNamespace(string @namespace)
         {
-            if (TryExecuteCommandWithVerboseLoggingOnly("get", "namespace", @namespace))
+            if (kubectl.ExecuteCommandWithVerboseLoggingOnly("get", "namespace", @namespace).ExitCode == 0)
                 return;
 
-            if (!TryExecuteCommandWithVerboseLoggingOnly("create", "namespace", @namespace))
+            if (kubectl.ExecuteCommandWithVerboseLoggingOnly("create", "namespace", @namespace).ExitCode != 0)
```

Comment:
This is where we now call the kubectl code, rather than the custom version in this class